src/Razor/test/Microsoft.VisualStudio.Editor.Razor.Test/TestFiles/ParserTests/DefaultVisualStudioRazorParserIntegrationTest/ImpExprAcceptsDCIInStmtBlkAfterIdentifiers_1.stree.txt

```diff
@@ -19,7 +19,7 @@ RazorDocument - [0..22)::22 - [@{LF    @DateTime..LF}]
                                     Transition;[@];
                                 CSharpImplicitExpressionBody - [9..19)::10
                                     CSharpCodeBlock - [9..19)::10
-                                        CSharpExpressionLiteral - [9..19)::10 - [DateTime..] - Gen<Expr> - ImplicitExpressionEditHandler;Accepts:NonWhitespace;ImplicitExpression[ATD];K21
+                                        CSharpExpressionLiteral - [9..19)::10 - [DateTime..] - Gen<Expr> - ImplicitExpressionEditHandler;Accepts:NonWhitespace;ImplicitExpression[ATD];K22
```

Comment:
It's really annoything that these numbers show up in the diffs. Next time this comes up we should nuke that from orbit.